web/crux/src/app/token/token.controller.ts

```diff
@@ -29,6 +30,7 @@ export default class TokenController implements CruxTokenController {
     return this.authService.getTokenList(request)
   }
 
+  @UseGuards(TokenDeleteAccessGuard)
```

Comment:
I believe we need this on the whole controller, what do you think?

Reply:
Except on the generate endpoint maybe?